corehq/apps/enterprise/urls.py

```diff
@@ -30,4 +35,6 @@
         name=ManageSSOEnterpriseView.urlname),
     url(r'^sso/(?P<idp_slug>[^/]*)/$', EditIdentityProviderEnterpriseView.as_view(),
         name=EditIdentityProviderEnterpriseView.urlname),
+
+    url(r'^interfaces/', include(interface_urls)),
```

Comment:
I'm not sure if this violates consistency with the rest of our code base, but I'd like to see a name change here; URLs are user-facing, and I don't think `interfaces` means anything to users.